Source/WebCore/style/StyleBuilder.cpp

```diff
@@ -114,6 +114,8 @@ void Builder::applyHighPriorityProperties()
 {
     applyProperties(firstHighPriorityProperty, lastHighPriorityProperty);
     m_state.updateFont();
+    // This needs to apply before other properties for the `lh` unit, but after updating the font.
+    applyProperties(CSSPropertyLineHeight, CSSPropertyLineHeight);
```

Comment:
Nothing here seems to prevent `line-height` from being applied a second time inside `applyNonHighPriorityProperties`. Why is that OK?

Reply:
There is a check here: https://searchfox.org/wubkat/rev/b36cbce69fddb7da33823f316bd8ead5bebee970/Source/WebCore/style/StyleBuilder.cpp#145